TramsDataApi/Migrations/TramsDb/20220627133721_NTI_UnderConsideration.Designer.cs

```diff
@@ -838,7 +838,9 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                     b.Property<string>("Author")
                         .HasColumnType("nvarchar(max)");
 
-                    b.Property<bool?>("BenefitsSectionIsCompleted")
+                    b.Property<bool?>("" +
```

Comment:
This is a bit weird.

Reply:
Strange it shows changes here, this is a migration from four months ago.